src/main/java/org/jabref/gui/libraryproperties/constants/ConstantsPropertiesViewModel.java

```diff
@@ -86,9 +86,12 @@ private ConstantsItemModel convertFromBibTexString(BibtexString bibtexString) {
 
     @Override
     public void storeSettings() {
-        databaseContext.getDatabase().setStrings(stringsListProperty.stream()
-                                                                    .map(this::fromBibtexStringViewModel)
-                                                                    .collect(Collectors.toList()));
+        List<BibtexString> strings = stringsListProperty.stream()
+                .map(this::fromBibtexStringViewModel)
+                .toList();
+        strings.forEach(string -> string.setParsedSerialization("@String{" +
```

Comment:
Can you extract this to a new method on the BibTexString class itself, e.g. call it serialize() because we have all information there already